lib/redis/interval/getter.ex

```diff
@@ -10,6 +10,15 @@ defmodule Castle.Redis.Interval.Getter do
     cache_hits(range, counts)
   end
 
+  def get_hits(key_prefix, ident, from, to, rollup) do
```

Comment:
The normal `get` method only returns data up until the first interval miss.  So if `1/21-1/22` and `1/24` are cached, and I asked for `1/21-1/24` ... I would only get those first 2 days, and then it would cache-miss on the last 2.  This method would instead return 3 days of data, with a hole in the middle.

(This is used by the total/trends rollups later, which just look for data in redis, and will never hit bigquery themselves).